index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Product landing page</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+
+    <header id="header">
+      <nav id="nav-bar">
+          <img id="header-img" class="logo" src="/samsung.png" alt="samsung-logo">
+          <ul class="nav-menu">
+            <li class="nav-link">
+              <a href="#video" target="_blank">Samsung phone</a>
+            </li>
+            <li class="nav-link">
+              <a href="http://samsung." target="_blank">Samsung tablets</a>
+            </li>
+            <li class="nav-link">
+              <a href="http://samsung." target="_blank">Samsung house</a>
+            </li>
+          </ul>
+      </nav>
+    </header>
+
+    <div class="video">
+      <iframe itemid="video" width="420" height="315"
+         src="https://www.youtube.com/embed/sq4scK7hgxo" id="video">Samsung phone products
+      </iframe>
+    </div>
+    <section>
+        <form id="form">
+          <label> Email</label> <br>
+          <input
+          id="email"
+          type="email"
+          required
+          placeholder="username@beststartupever.com"
+          pattern=".+@beststartupever\.com"
```

Comment:
This is a polite Suggestion. You can use HTML semantic tags as they give meaning to the content. Also, please avoid overusing IDs, as they are unique. This means that if you attach a style to an ID, you won't be able to reuse it within the same webpage. Classes, however, can appear on several HTML elements on the same page.